lib/http.js

```diff
@@ -97,6 +97,52 @@ HttpClient.prototype.handleResponse = function(req, res, body) {
   return body;
 };
 
+/**
+ * Handle the http Attachment response
+ * @param {Object} The req object
+ * @param {Object} res The res object
+ * @param {Object} body The http body
+ * @param {Object} The attachment List
+ */
+HttpClient.prototype.handleAttachmentResponse = function(req, res, body) {
+  debug('Http response body: %j', body);
+  var arr = [];
+  if (typeof body === 'string') {
+     if ((res !== undefined)&&(res !== null)&&(res.headers['content-type'] !== undefined)&&(res.headers['content-type'] !== null)) {
+       var boundary = res.headers['content-type'].match(/boundary=\"([^\"]*)\"/i);
+       
+       if (boundary !== null){
```

Comment:
By writing`if (!boundary)` instead of `if (boundary !== null)` ou also check that the variable is not undefined